cmd/config/init/unique_rollapp_id.go

```diff
@@ -0,0 +1,32 @@
+package initconfig
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func isRollappIDUnique(rollappID string) (bool, error) {
+	url := HubData.API_URL + "/dymensionxyz/dymension/rollapp/rollapp/" + rollappID
```

Comment:
in case this url will be used again I suggest taking it out to a const name `rollapp_endpoint` or something along those lines.